src/main/java/io/nats/client/impl/NatsConnection.java

```diff
@@ -718,11 +722,11 @@ NatsSubscription createSubscription(String subject, String queueName, NatsDispat
         sub = new NatsSubscription(sid, subject, queueName, this, dispatcher);
         subscribers.put(sid, sub);
 
-        sendSubscriptionMessage(sid, subject, queueName);
+        sendSubscriptionMessage(sid, subject, queueName, false);
         return sub;
     }
 
-    void sendSubscriptionMessage(CharSequence sid, String subject, String queueName) {
+    void sendSubscriptionMessage(CharSequence sid, String subject, String queueName, boolean fastPath) {

```

Comment:
Not sure of the naming.. not really representative of what happens. `useInternalOutgoing` may make more sense ;-)